microsoft-edge/webdriver-chromium/ie-mode.md

```diff
@@ -0,0 +1,222 @@
+
+            InternetExplorerDriver reqDriver = 
+                                   new InternetExplorerDriver(ieService, ieOptions);
+            reqDriver.Url = "https://bing.com";
+            reqDriver.FindElementById("sb_form_q").SendKeys("WebDriver");
+            reqDriver.FindElementById("sb_form").Submit();
+
+            Console.WriteLine("Done");
+        }
+    }
+}
+```
+
+### [Python](#tab/python/)
+
+<a id="drive-ie-mode-entire-sample"></a>
+
+<!-- todo -->
+Insert Python sample here
+
+* * *
+
+The following sections explain the steps in this sample in more detail.
+
+
+<!-- ====================================================================== -->
+## Create an InternetExplorerDriverService
+
+### [C#](#tab/c-sharp/)
+
+<a id="drive-ie-mode-ieService"></a>
+
+To create an `InternetExplorerDriverService`, use the `CreateDefaultService` method.  Pass in the following parameters:
+
+*  `dir`, which is the directory where you downloaded IEDriver to.
+*  `driverexe`, which is the name of the IEDriver executable.
+
+```csharp
+//change the path accordingly
+var dir = @"C:\<path to IEDriver download>";
+var driverexe = "IEDriverServer.exe";
+
+var ieService = InternetExplorerDriverService.CreateDefaultService(dir, driverexe);
+```
+
+### [Python](#tab/python/)
+
+<a id="drive-ie-mode-ieService"></a>
+
+<!-- todo -->
+Insert Python sample here
+
+* * *
+
+
+<!-- ====================================================================== -->
+## Define InternetExplorerOptions with additional capabilities for Microsoft Edge
+
+### [C#](#tab/c-sharp/)
+
+<a id="drive-ie-mode-ieOptions"></a>
+
+1. Define a new variable, `ieOptions`, by instantiating `InternetExplorerOptions` with the `IgnoreZoomLevel` property set to `true`.
+
+1. Call the `AddAdditionalCapability` method, with the `ie.edgechromium` property set to `true`, and `ie.edgepath` set to the path of the Microsoft Edge executable:
+
+```csharp
+var ieOptions = new InternetExplorerOptions { IgnoreZoomLevel = true };
+ieOptions.AddAdditionalCapability("ie.edgechromium", true);
+//change the path accordingly
+ieOptions.AddAdditionalCapability("ie.edgepath", 
+       "C:/Program Files (x86)/Microsoft/Edge/Application/msedge.exe");
+```
+
+### [Python](#tab/python/)
+
+<a id="drive-ie-mode-ieOptions"></a>
+
+<!-- todo -->
+Insert Python sample here
+
+* * *
+
+
+<!-- ====================================================================== -->
+## Start InternetExplorerDriver
+
+### [C#](#tab/c-sharp/)
+
+<a id="drive-ie-mode-ieDriver"></a>
+
+Finally, start `InternetExplorerDriver` and pass it the previously defined `ieService` and `ieOptions`.  IEDriver launches Microsoft Edge in IE mode.  All page navigation and subsequent interactions occur in IE mode.
+
+```csharp
+InternetExplorerDriver reqDriver = new InternetExplorerDriver(ieService, ieOptions);
+```
+
+### [Python](#tab/python/)
+
+<a id="drive-ie-mode-ieDriver"></a>
```

Comment:
```suggestion
<a id="drive-ie-mode-iedriver"></a>
```
or ie-driver.  Might check Docs Contributor Guide re: lowercasing the id attribute.